src/garnet/logger.py

```diff
@@ -0,0 +1,17 @@
+"""Logger for GARNET"""
+
+from typing import Any
+
+from mantid.kernel import Logger as MantidLogger
+
+
+class Logger(MantidLogger):
+    """Logger class for GARNET
+
+    This is a thin wrapper for the Mantid logger class. It adds the ability to
+    get the garnet logger in a simple way.
+    """
+
+    def get(self: Any, name: str = "garnet"):
```

Comment:
Since you already have type annotations, the signature could be
```python
def get(self: Any, name: str = "garnet") -> garnet.logger.Logger:
```
or maybe it is actually a `mantid.kernel.Logger`